sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveExternalCatalogVersionsSuite.scala

```diff
@@ -195,7 +195,7 @@ class HiveExternalCatalogVersionsSuite extends SparkSubmitTestUtils {
 
 object PROCESS_TABLES extends QueryTest with SQLTestUtils {
   // Tests the latest version of every release line.
-  val testingVersions = Seq("2.0.2", "2.1.2", "2.2.0")
+  val testingVersions = Seq("2.0.2", "2.1.2", "2.2.0", "2.2.1")
```

Comment:
Please don't mix this into a PR with title `Document`.

Reply:
This is to verify what we explained in doc is correct.

Reply:
Yep. Please update the title to include that, too.

Reply:
Already has it in the PR description.

Reply:
The main goal of this PR is not to test it but to document it. We need another backport PR to SPARK 2.2 without including the test.